conf/application.conf

```diff
@@ -18,7 +18,7 @@ appName="manage-transit-movements-frontend"
 
 play.http.router=prod.Routes
 
-play.filters.headers.contentSecurityPolicy = "default-src 'self' 'unsafe-inline' localhost:9485 localhost:9032 localhost:9250 localhost:12345 www.google-analytics.com data: https://www.googletagmanager.com https://fonts.googleapis.com tagmanager.google.com https://ssl.gstatic.com https://www.gstatic.com https://fonts.gstatic.com https://fonts.googleapis.com"
+play.filters.headers.contentSecurityPolicy = "default-src 'self' 'unsafe-inline' localhost:9485 localhost:9032 localhost:9250 localhost:12345 www.google-analytics.com www.googletagmanager.com tagmanager.google.com www.google-analytics.com"
```

Comment:
Is this a duplicate whitelist? `www.google-analytics.com`

Reply:
Yes it is, nice spot Paul